src/test/java/uk/gov/pay/api/service/CreatePaymentServiceTest.java

```diff
@@ -189,4 +189,24 @@ public void creating_payment_with_moto_when_not_allowed_for_account_should_retur
             assertThat(e.getErrorIdentifier(), is(ErrorIdentifier.MOTO_NOT_ALLOWED));
         }
     }
+
+    @Test
+    @PactVerification({"connector"})
+    @Pacts(pacts = {"publicapi-connector-create-payment-with-credentials-in-created-not-allowed"})
+    public void creating_payment_with_gateway_account_credentials_in_created_state_should_return_400() {
```

Comment:
I think test could be named to match the exception being unit tested. May be something like below ?
```suggestion
    public void shouldThrowExceptionWithIdentifierAccountNotLinkedToPSP_IfGatewayAccountCredentialIsNotConfiguredInConnector() {
```

Reply:
Done.